apps/db/priv/repo/migrations/20220126101800_add_title_resource_history.exs

```diff
@@ -0,0 +1,24 @@
+defmodule DB.Repo.Migrations.AddTitleResourceHistory do
+  use Ecto.Migration
+
+  def up do
+    # Add the key `title` to the JSON payload and set it
+    # to the resource's current title
+    execute """
+    update resource_history
+    set payload = jsonb_set(payload, '{title}', to_jsonb(t.title))
```

Comment:
Pour les électeurs: https://dev.to/deepika_banoth/how-to-use-jsonbset-function-in-postgresql-35eo